docs/AppBuildAndDeploy.md

```diff
@@ -248,12 +253,11 @@ extends:
             filepath: "./appConfig"         #Optional: Folder path of app configuration files
             filetype: "yaml"                #Optional: default value json
             variableGroups:                 #Optional: List of variable groups which contain secrets
-              - variableGroup1              #Optional: Variable Group name 
-              - variableGroup2      
-            variables:                      #Optional: List of variables used by the service
-              - servicename-variable1
-              - servicename-variable2
-
+              - variableGroup<environment>  #Optional: Variable Group name, if specified with <environment>, relevant env specific variables will be imported
```

Comment:
variableGroups (plural.. ?)

Reply:
In line 256 it is singular, because it is a list.  Ex  - ffc-demo-web-<environment>